create-deploy-pr/src/index.ts

```diff
@@ -119,6 +119,43 @@ async function createDeployPr(deployData: DeploymentData, targetRepo: RepoData,
     }
 }
 
+async function createDeployCommit(deployData: DeploymentData, targetRepo: RepoData, baseRepoName: string,octokit: any): Promise <void>{
+
+    console.log('Get deployment config map content');
+    //Get deployment config map content
+    const { data: cmData} = await octokit.repos.getContent({
+        owner: targetRepo.repoOrg,
+        repo: targetRepo.repoName,
+        ref: `refs/heads/${targetRepo.branchName}`,
+        path: deployData.cmPath
+    });
+
+    let content = Buffer.from(cmData.content, 'base64').toString();
+    //Set new values in deployment config map
+    let deployContent = setConfigMap(deployData.key, deployData.keyValue, content);
+
+    console.log('Push deployment config map content to target directory');
+    //Push deployment config map content to target directory
+    const { data: cmResult } = await octokit.repos.createOrUpdateFileContents({
+        owner: targetRepo.repoOrg,
+        repo: targetRepo.repoName,
+        path: deployData.cmPath,
+        branch: targetRepo.branchName,
+        content: Buffer.from(deployContent).toString("base64"),
+        sha: cmData.sha,
+        message: `Automated update ${baseRepoName}`,
+        committer:{
+            name: 'vc-ci',
+            email: 'ci@virtocommerce.com' 
+        },
+        author:{
+            name: 'vc-ci',
+            email: 'ci@virtocommerce.com' 
```

Comment:
Shall we move these values to parameters with default values?

Reply:
Good idea!